core/assets/localization/default.properties

```diff
@@ -98,7 +98,7 @@ editor.msg.selectionHint.count.add={0} will be added to the selection
 editor.msg.multipartSplit=Left click to split entities with multiple cues
 editor.msg.cannotSplit=[RED]Cannot split entity[] because the split cues would intersect with other cues
 editor.msg.timeSignature=Left/right click to place/delete time signature changes - Scroll to change beats per measure (Hold [CYAN]CTRL[] for ±5) - Hold [CYAN]SHIFT[] and scroll to change beat divisions
-editor.msg.tempoChange=Left/right click to place/delete tempo changes - Drag the ends for a gradual transition  - Scroll to change - Hold [CYAN]CTRL[] for ±5, [CYAN]SHIFT[] for ±0.1
+editor.msg.tempoChange=Left/right click to place/delete tempo changes - Hold [CYAN]ALT[] while clicking to double tempo, hold [CYAN]SHIFT[] to halve it - Drag the ends for a gradual transition - Scroll to change - Hold [CYAN]CTRL[] for ±5, [CYAN]SHIFT[] for ±0.1
```

Comment:
"Clicking" is ambiguous. My recommended wording is "Hold [CYAN]ALT[] while placing to double the tempo, [CYAN]SHIFT[] to halve it".